modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -992,6 +993,35 @@ void TFImporter::populateNet(Net dstNet)
             if (getDataLayout(name, data_layouts) == DATA_LAYOUT_UNKNOWN)
                 data_layouts[name] = DATA_LAYOUT_NHWC;
         }
+        else if (type == "MirrorPad")
```

Comment:
Please try to reduce code duplication. Try to integrate this padding mode into existing zero padding.